test/sys/test_signal.rs

```diff
@@ -5,3 +5,41 @@ use nix::sys::signal::*;
 fn test_kill_none() {
     kill(getpid(), None).expect("Should be able to send signal to myself.");
 }
+
+#[test]
+fn test_sigprocmask_noop() {
+    sigprocmask(SigmaskHow::SIG_BLOCK, None, None)
+        .expect("this should be an effective noop");
+}
+
+#[test]
+fn test_sigprocmask() {
```

Comment:
You should grab `SIGNAL_MTX` here so that this doesn't race with any other tests.